### ISSUE TITLE
Major entity bloom for non-glowing entities

### ISSUE THREAD

**Issue by RecursivePineapple** [GTNewHorizons/Angelica#816]
### Your GTNH Discord Username

recursive_pineapple

### Mod Version

1.0.0-beta27

### Bug Report

Whenever I look in a specific direction (always ~45 degrees down, but the cardinal direction changes) most mobs have severe bloom. From what I can see it's somewhat related to the time of day/moon phase, but I can't tell what the exact connection is.

GPU: 3060ti

Shaders: complementary reimagined 5.3
Shader settings:
```
#Fri Dec 20 23:56:51 EST 2024
BLOCK_REFLECT_QUALITY=1
FXAA_DEFINE=-1
LIGHTSHAFT_QUALI_DEFINE=1
NIGHT_NEBULA=true
RP_MODE=3
SHADOW_QUALITY=1
shadowDistance=128.0
```

![2024-12-20_23 52 50](https://github.com/user-attachments/assets/c285b0da-168b-4112-97d2-ccb566ce890d)
![2024-12-21_00 00 04](https://github.com/user-attachments/assets/5506eccb-bae8-488a-888b-4573c4a2ea2c)

### Java Version

Java 21

### Graphics Card Vendor

NVidia

### Shaders

Yes: Please put relevant information about shaderpack in the report(what shaderpack, any special settings of note, etc)

### MCPF or other features disabled by default

No: Go ahead and create an issue

### Mod List or GTNH Pack Version

GTNH 2.7.1

### Final Checklist

- [X] I have searched the issues and haven't found a similar issue.
- [X] I have read the known incompatibilities and this is not related to one of those.
- [X] I am running an officially released version.  (Or, if I've compiled it myself I plan to fix the issue)

**Comment by RecursivePineapple** [comment]
Looks like this only happens with labPBR/seuspbr (which I enabled without knowing what it was).

**Comment by calreveraster** [comment]
No, this happens on certain other mods that arent labPBR. For example, if you're holding an adventurebackpack, it happens there too: 

![image](https://github.com/user-attachments/assets/111eba76-d106-4b05-8c81-eeefd2611836)

**Comment by MalTeeez** [comment]
Seems to be fixed on latest:

<img width="1201" height="980" alt="Image" src="https://github.com/user-attachments/assets/a93d900f-ad0f-4a59-8ca8-19460888f5fc" />

<img width="1914" height="674" alt="Image" src="https://github.com/user-attachments/assets/3a5ab289-c1d8-4e21-a533-6aa53df7777b" />
